### PR TITLE
Add Apache Solr example.

### PR DESCRIPTION
**Pull request by mnagaya**
We have this config running in production.

### CODE REVIEW

File: example_configs/solr.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+whitelistObjectNames: [ "java.lang:*", "org.eclipse.jetty.util.thread:*", "org.eclipse.jetty.server.handler:*", "solr/*:type=/select,*", "solr/*:type=/update,*", "solr/*:type=searcher,*", "solr/*:type=documentCache,*", "solr/*:type=updateHandler,*"]
+
+rules:
+  - pattern: "^java(.(?!OperatingSystem))*$"
```
Comment (brian-brazil) [inline example_configs/solr.yml]: Exclude all these, they're already taken care of by the default exports

File: example_configs/solr.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+whitelistObjectNames: [ "java.lang:*", "org.eclipse.jetty.util.thread:*", "org.eclipse.jetty.server.handler:*", "solr/*:type=/select,*", "solr/*:type=/update,*", "solr/*:type=searcher,*", "solr/*:type=documentCache,*", "solr/*:type=updateHandler,*"]
```
Comment (brian-brazil) [inline example_configs/solr.yml]: Why are you adding this?

File: example_configs/solr.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+rules:
+  - pattern: "^java(.(?!OperatingSystem))*$"
+  - pattern: "^solr/([a-zA-Z0-9_]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+)<type=(.+), id=(.+)><>(.*):"
+    name: solr_$4_$6
+    labels:
+      collection: $1
+      shard: $2
+      replica: $3
```
Comment (brian-brazil) [inline example_configs/solr.yml]: Does this happen to be a hostname?

File: example_configs/solr.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  - pattern: ".*"
```
Comment (brian-brazil) [inline example_configs/solr.yml]: You shouldn't fall back to the defaults here, instead create rules for what you want.

**Comment by mnagaya** [comment]
Sorry for late reply.  Solr7.6 now provides solr-exporter:
https://lucene.apache.org/solr/guide/7_6/monitoring-solr-with-prometheus-and-grafana.html

**Pull request closed without merging**